vid_sdl2.c

```diff
@@ -562,6 +568,11 @@ static void VID_SDL_Init(void)
 		}
 	}
 
+	SDL_SetHint(SDL_HINT_VIDEO_MAC_FULLSCREEN_SPACES, "0");
```

Comment:
Yes it should, I put a TODO on it, I can't commit from work so I asked @fzwoch to do the patch. However that line went on the wrong side of the ifdef and he was too lazy to fix it :)